torchvision/prototype/transforms/functional/_dispatch.py

```diff
@@ -0,0 +1,448 @@
+# THIS FILE IS AUTOGENERATED
```

Comment:
As the line states, this is the autogenerated file. Note that if we don't autogenerate this. we would still have exactly the same implementation, but written manually.